sdk/eventhub/azure-mgmt-eventhub/tests/_aio_testcase.py

```diff
@@ -23,12 +23,12 @@ def create_mgmt_aio_client(self, client, **kwargs):
             from azure.identity.aio import DefaultAzureCredential
             credential = DefaultAzureCredential()
         else:
-            credential = Mock(get_token=lambda _: self.mock_completed_future(AccessToken("fake-token", 0)))
+            credential = Mock(spec_set=["get_token"], get_token=lambda _: self.mock_completed_future(AccessToken("fake-token", 0)))
```

Comment:
just checking, there wasnt a sync counterpart for EH ?

Reply:
Nope, and it's not really used from what I can tell since the majority of mgmt packages tests are disabled (including all EH mgmt tests). I only updated here for posterity.